libclamav/scanners.c

```diff
@@ -2045,6 +2045,10 @@ static cl_error_t cli_ole2_tempdir_scan_vba(const char *dir, cli_ctx *ctx, struc
         free(fullname);
     }
 
+    if (fd >= 0) {
```

Comment:
shouldn't this be > 2, since 0 is stdin, 1 is stdout, and 2 is stderr?

Reply:
It's possible to close stdin, stdout, and stderr in which case the 0-2 are valid file descriptors for files.  Not that we do that, as far as I know.  But perhaps some application using libclamav might?